release/public-v2: addd RRFs v1alpha regression tests

climbfuji opened the pull request:
## Description

- add regression tests for new suite FV3_RRFS_v1alpha
- remove redundant `effr_in=.true.` from namelist templates
- change `NEW_BASELINE` in rt.sh to avoid conflicts when creating new baselines for release/public-v2 and develop at the same time

## Testing

Regression tests were run on the following platforms as for the previous release/public-v2 PRs: cheyenne.{gnu,intel}, hera.{gnu,intel}, jet.intel, gaea.intel. Logs updated in the PR.

## Dependencies

https://github.com/NCAR/ccpp-physics/pull/518
https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/61
https://github.com/NOAA-EMC/fv3atm/pull/197
https://github.com/ufs-community/ufs-weather-model/pull/276

mkavulich reviewed (approved):
Looks good to me. Once it is merged we will update the SRW app to point to the latest hash so we can use the new suite in the release branch.

climbfuji commented:
> Looks good to me. Once it is merged we will update the SRW app to point to the latest hash so we can use the new suite in the release branch.

These PRs are waiting for the approval and merge of the GFDL_atmos_cubed_sphere PR https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/61 - everything is ready to go.

junwang-noaa commented:
Dom, who would you recommend to review the dycore PR#61?

On Mon, Nov 16, 2020 at 10:17 PM Dom Heinzeller <notifications@github.com>
wrote:

> Looks good to me. Once it is merged we will update the SRW app to point to
> the latest hash so we can use the new suite in the release branch.
>
> These PRs are waiting for the approval and merge of the
> GFDL_atmos_cubed_sphere PR NOAA-GFDL/GFDL_atmos_cubed_sphere#61
> <https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/61> -
> everything is ready to go.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/276#issuecomment-728659754>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI7D6TI7GZNRPUR7XSWQLOLSQHTMNANCNFSM4TT45ZPQ>
> .
>

climbfuji commented:
> Dom, who would you recommend to review the dycore PR#61?
> […](#)

Anyone from GFDL will be fine. I asked Rusty to review, but so far no reply. Note that this PR is for the NOAA-GFDL fork, not the NOAA-EMC fork (because it is for the public release branch) - can you assign reviewers and merge the code there?

junwang-noaa commented:
No, I can't. Only GFDL code managers can do that.

On Tue, Nov 17, 2020 at 9:21 AM Dom Heinzeller <notifications@github.com>
wrote:

> Dom, who would you recommend to review the dycore PR#61?
> … <#m_-601748604721204011_>
>
> Anyone from GFDL will be fine. I asked Rusty to review, but so far no
> reply. Note that this PR is for the NOAA-GFDL fork, not the NOAA-EMC fork
> (because it is for the public release branch) - can you assign reviewers
> and merge the code there?
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/276#issuecomment-728959551>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI7D6TN3A2UN3NDU2LSDZ23SQKBGJANCNFSM4TT45ZPQ>
> .
>

climbfuji commented:
I updated the submodule pointer for fv3atm and reverted the change to `.gitmodules`. Since the PR has been approved by the code managers, I can merge it myself - will do so.
